py/selenium/webdriver/ie/options.py

```diff
@@ -387,4 +387,4 @@ def to_capabilities(self) -> dict:
 
     @property
     def default_capabilities(self) -> dict:
-        return DesiredCapabilities.INTERNETEXPLORER.copy()
\ No newline at end of file
+        DesiredCapabilities.INTERNETEXPLORER.copy()
```

Comment:
Has Python now moved to implicit returns?

Reply:
OMG... just unbelievable  :) while fixing lynting issue somehow I missed `return` statement... Thank you so much for letting me know...